core/gui/src/app/workspace/component/input-autocomplete/input-autocomplete.component.scss

```diff
@@ -1,3 +1,17 @@
 mat-form-field {
   width: 100%;
 }
+.input-autocomplete-container {
+  display: flex;
+  align-items: center;
+  width: 100%;
+
+  input {
+    flex: 1;
+    margin-right: 10px;
+  }
+
+  button {
```

Comment:
I remember @aglinxinyuan mentioned during our group discussion that the button color looks quite similar to the 'Run' button. It was suggested that we could make the button's border blue instead of coloring the entire button. What are your thoughts on this suggestion?

Reply:
yes, I made some changes.
<img width="315" alt="截屏2024-09-18 下午2 21 24" src="https://github.com/user-attachments/assets/02a183a1-a4ef-4b8b-bf48-de6a5040e670">
how is this design?

Reply:
Looks good!